src/commands/build/watch/mod.rs

```diff
@@ -0,0 +1,86 @@
+mod watcher;
+pub use watcher::wait_for_changes;
+
+use crate::commands::build::{command, wranglerjs};
+use crate::commands::publish::Package;
+use crate::settings::project::{Project, ProjectType};
+use crate::terminal::message;
+use crate::{commands, install};
+
+use notify::{watcher, RecursiveMode, Watcher};
+use std::env;
+use std::sync::mpsc::{channel, Sender};
```

Comment:
same here: i would `use std::sync` so it's clear that channel and Sender are from mpsc